homeassistant/components/roon/config_flow.py

```diff
@@ -0,0 +1,114 @@
+"""Config flow for roon integration."""
+from asyncio import sleep
+import logging
+
+from roon import RoonApi
+import voluptuous as vol
+
+from homeassistant import config_entries, core, exceptions
+from homeassistant.const import CONF_API_KEY, CONF_HOST
+
+from .const import DEFAULT_NAME, DOMAIN, ROON_APPINFO  # pylint: disable=unused-import
+
+_LOGGER = logging.getLogger(__name__)
+
+DATA_SCHEMA = vol.Schema({"host": str})
+
+TIMEOUT = 120
+
+
+class RoonHub:
+    """Interact with roon during config flow."""
+
+    def __init__(self, host):
+        """Initialize."""
+        self._host = host
+
+    async def authenticate(self, hass) -> bool:
+        """Test if we can authenticate with the host."""
+        token = None
+        secs = 0
+        try:
+            roonapi = RoonApi(ROON_APPINFO, None, self._host, blocking_init=False)
+            while secs < TIMEOUT:
+                token = roonapi.token
+                secs += 5
+                if token:
+                    break
+                await sleep(5, hass.loop)
```

Comment:
Don't pass the loop. That's deprecated.
```suggestion
                await asyncio.sleep(5)
```